.storybook/main.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   env: (config) => ({
     ...config,
     REACT_APP_BACKEND_URL: process.env.REACT_APP_BACKEND_URL,
+    REACT_APP_PASSWORD_SECRET_KEY: "blahblahblahblah",
```

Comment:
can remove this now right?

Reply:
yup, https://github.com/isomerpages/isomercms-frontend/pull/1316/commits/d7782a3614511cb981589a03759f25fbadcbbce3